crates/tasks/src/lib.rs

```diff
@@ -251,15 +251,35 @@ impl TaskExecutor {
     }
 
     /// Spawns a future on the tokio runtime depending on the [TaskKind]
-    fn spawn_on_rt<F>(&self, fut: F, task_kind: TaskKind) -> JoinHandle<()>
+    ///
+    /// With `tokio-console` feature flag enabled, the tasks are spawned through an unstable task
+    /// builder. The returned `Result` is currently unreachable. Ref: <https://docs.rs/tokio/latest/src/tokio/task/builder.rs.html#91>.
+    fn spawn_on_rt<F>(&self, name: &'static str, fut: F, task_kind: TaskKind) -> JoinHandle<()>
```

Comment:
Imo better to have an `Option` for name and name the tasks that are currently named "" - I'm unsure on the behavior of tokio in this case, would everything just be named ""? If we don't pass a name, I assume something low-context is chosen for us

Reply:
its more "rusty" to use Option instead of `""` lol